digdag-tests/src/test/java/acceptance/RequireIT.java

```diff
@@ -115,6 +118,27 @@ public void testRequire()
 
         // Verify that the file created by the child workflow is there
         assertThat(Files.exists(childOutFile), is(true));
+
+        CommandStatus attemptTasks = main("tasks",
+                "-c", config.toString(),
+                "-e", server.endpoint(),
+                String.valueOf(attemptId));
```

Comment:
May be `attemptId.toString()` or `attemptId.get()` is enough ?

Reply:
Agreed. This code follows the existing style.
https://github.com/treasure-data/digdag/blob/c9787276b8d3332da9ed3a87aaf330a222d44df9/digdag-tests/src/test/java/acceptance/RequireIT.java#L107

I'm going to fix my code.

Reply:
Updated https://github.com/hkdnet/digdag/commit/19de2e155de25ddd34f1e45e75c40457567bac36